models/Sherlock-Benchmark-9-TORA/Sherlock-Benchmark-9-TORA.jl

```diff
@@ -50,6 +50,8 @@ const verification = false;
 end
 
 path = @modelpath("Sherlock-Benchmark-9-TORA", "controllerTora.mat")
+# path = @modelpath("Sherlock-Benchmark-9-TORA", "controllerToraSigmoid.mat")
```

Comment:
Create a copy of the TORA benchmark with one of these controllers (to catch the bug).